src/include/btree.i

```diff
@@ -979,6 +980,13 @@ __wt_page_can_evict(WT_SESSION_IMPL *session, WT_PAGE *page, int check_splits)
 		return (0);
 
 	/*
+	 * Allow for the splitting of pages when a checkpoint is underway
+	 */
+	if (allow_splits && page->memory_footprint > btree->maxmempage &&
+		    btree->checkpointing && !F_ISSET(mod, WT_PM_REC_MULTIBLOCK))
```

Comment:
This spacing is wrong. Indent to plus 4 characters.

It'd be good if the comment explained why the MULTIBLOCK part of the condition is there.